python-packages/smithy-core/tests/unit/test_types.py

```diff
@@ -1,5 +1,6 @@
 #  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 #  SPDX-License-Identifier: Apache-2.0
+# pyright: reportPrivateUsage=false
```

Comment:
Does this only apply to this file for checks? It also seems odd to have this connected to the Copyright block. Is the formatter pushing the two together?

Reply:
Yeah, this is a file-wide check. I can separate by an empty line and it should be fine.